tests/components/co2signal/test_config_flow.py

```diff
@@ -22,7 +26,7 @@ async def test_form_home(hass: HomeAssistant) -> None:
     assert result["errors"] is None
 
     with patch(
-        "CO2Signal.get_latest",
+        "aioelectricitymaps.ElectricityMaps._get",
```

Comment:
Can we patch the public function we are calling? And I know this didnt happen before, but can we patch it at the place where its used? `homeassistant.components.co2signal.config_flow.ElectricityMaps.....`

Reply:
Yes, this should be possible, but I wanted to keep the tests as original as possible for this PR. I can change this in a later PR. 

The "patch place" will change in a second... or two.... :)

Reply:
haha great